src/util/make-periodic-task.ts

```diff
@@ -0,0 +1,61 @@
+export interface PeriodicTask {
```

Comment:
Since this is based of the one in the core, you might want to take a look at the changes to the core's version to allow the task to be created but only started after the first time interval: https://github.com/EdgeApp/edge-core-js/blob/master/src/util/periodic-task.js

Reply:
True, but that isn't relevant to this PR. Perhaps a task to update this utility should be created for this purpose.